src/components/Cards.js

```diff
@@ -0,0 +1,20 @@
+import Card from './Card';
+import './Cards.css'
+function Cards(props) {
+    return (
+        <div className='width-100 column align-center cards'>
+        {props.categories.map((category,categoryIndex)=>{
+            return(
+                <div className='column align-base default-color category'>
+                    <h2>{category}</h2>
+                    {props.cardsData.filter( card => card.category === category).map((cardData,index)=>{
```

Comment:
Move this section to a function, and name it something like `renderCards` and call it here by passing the arguments